assets/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Wäre evtl möglich das ins root zu ziehen damit alles aus einem Ordner raus machbar ist?

Reply:
Ja klar. Kann ich umstellen.

Reply:
Hab das gerade umgebaut. `package*json` liegt jetzt im Root-Verzeichnis. Keine Notwendigkeit mehr für Prefixes. Hatte gerade auch noch das Problem, dass die generierte assets.go im Dev-Mode nicht mit neu erzeugten Dateien klargekommen ist. Daher habe ich jetzt ne explizite Fallunterscheidung in die `http.go` eingebaut. Schau dir mal bitte an, ob man das so machen kann.